Doc/library/sqlite3.rst

```diff
@@ -1345,27 +1347,23 @@ The following example illustrates the implicit and explicit approaches:
 .. literalinclude:: ../includes/sqlite3/converter_point.py
 
 
-Default adapters and converters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There are default adapters for the date and datetime types in the datetime
-module. They will be sent as ISO dates/ISO timestamps to SQLite.
+Default adapters and converters (deprecated)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The default converters are registered under the name "date" for
-:class:`datetime.date` and under the name "timestamp" for
-:class:`datetime.datetime`.
-
-This way, you can use date/timestamps from Python without any additional
-fiddling in most cases. The format of the adapters is also compatible with the
-experimental SQLite date/time functions.
+.. note::
 
-The following example demonstrates this.
+   The default adapters and converters are deprecated as of Python 3.12.
+   Instead, use the :ref:`sqlite3-adapter-converter-recipes`,
+   and tailor them to your needs.
 
-.. literalinclude:: ../includes/sqlite3/pysqlite_datetime.py
+The deprecated default adapters and converters consists of:
 
-If a timestamp stored in SQLite has a fractional part longer than 6
-numbers, its value will be truncated to microsecond precision by the
-timestamp converter.
+* Adapt :class:`datetime.date` objects to :class:`strings <str>` in ISO format.
+* Adapt :class:`datetime.datetime` objects to strings ISO format.
+* Convert :ref:`declared <sqlite3-converters>` "date" types to ``datetime.date``
+  objects.
+* Convert declared "timestamp" types to ``datetime.datetime`` objects.
```

Comment:
```suggestion
* An adapter for :class:`datetime.date` objects to :class:`strings <str>` in
  `ISO 8601 <https://en.wikipedia.org/wiki/ISO_8601>`_ format.
* An adapter for :class:`datetime.datetime` objects to strings in ISO 8601 format.
* A converter for :ref:`declared <sqlite3-converters>` "date" types to
  :class:`datetime.date` objects.
* A converter for declared "timestamp" types to :class:`datetime.datetime` objects.
```

* Explicitly specify and link ISO 8601 for clarity
* Fix grammar/agreement (tip: try inserting the bullet items after the colon—do they make grammatical sense as a sentence? E.g. "The adapters and converters consist of: adapt date objects to strings in ISO format." vs. "The adapters and converters consist of: an adapter for date objects to strings in ISO format.")
* Fix typo (missing word) in second item
* Use consistent `:class:` role/formatting for classes